src/14_cal.py

```diff
@@ -29,4 +29,50 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+
+def get_calendar(args):
```

Comment:
This is really well-done. While you weren't _required_ to validate input, it's great that you did. And knowing when and how to break problems up into smaller sub-functions is going to be really useful.